### PR TITLE
i18n: DT EX2 Trial, raids, ihuykatumu CN tranlation

### PR DESCRIPTION
**Pull request by YuzukiTsuru**

### CODE REVIEW

File: ui/raidboss/data/07-dt/dungeon/ihuykatumu.ts [hunk 1]
```diff
@@ -42,6 +43,7 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: 'Stay out of inhale',
           de: 'Steh auserhalb des Soges',
+          cn: '呆在AoE外面',
```
Comment (MaikoTan) [inline ui/raidboss/data/07-dt/dungeon/ihuykatumu.ts:46]:
```suggestion
          cn: '躲开 Boss 吸气 + 全场 AoE',
```

**Review by MaikoTan — COMMENTED** [review]
Minor comments

File: ui/raidboss/data/07-dt/raid/r1n.ts [hunk 2]
```diff
@@ -212,10 +212,12 @@ const triggerSet: TriggerSet<Data> = {
         cardsIntercards: {
           en: 'Cards => Intercards',
           de: 'Karten => Interkardinal',
+          cn: 'X字 => +字',
         },
         intercardsCards: {
           en: 'Intercards => Cards',
           de: 'Interkardinal => Karten',
+          cn: '+字 => X字',
```
[the remark this answers is not in the document]
Comment (MaikoTan) [reply ui/raidboss/data/07-dt/raid/r1n.ts:220]: No, I think you'd better use the `tts` property like:

```js
tts: {
  cn: '十字散开 => 叉字散开',
},

[the remark this answers is not in the document]
Comment (Echoring) [reply ui/raidboss/data/07-dt/raid/r1n.ts:220]: Here is no spread. An attack first cross then intercard or vice versa.

[the remark this answers is not in the document]
Comment (Echoring) [reply ui/raidboss/data/07-dt/raid/r1n.ts:220]: I take a closer look. This is calling safe point, I think `斜角 => 正边` would be better.

[the remark this answers is not in the document]
Comment (MaikoTan) [reply ui/raidboss/data/07-dt/raid/r1n.ts:220]: Just notating that you could differentiate the text for vision and auditory by different keys for a better UX.

**Comment by MaikoTan** [comment]
I would like to merge this before #264 so that you could avoid resolving git conflicts. @Echoring is it good to you?

**Review by Echoring — APPROVED** [review]

**Pull request merged**